src/connect_2020/azext_connect_2020/vendored_sdks/version.py

```diff
@@ -5,5 +5,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "2020-01-01-preview"
```

Comment:
@akashkeshari , Do this PR has your code changes with new swagger generated code?. 

Not sure if it's possible or not (you can do it from the next time as we don't have time now) but it's advised to split your changes and swagger changes into two PR's so that it would be easier for reviewers to review.